test/Microsoft.AspNet.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -0,0 +1,312 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    public class BinderTypeBasedModelBinderIntegrationTest
+    {
+        // The NullModelBinder and NullModelNotSetModelBinder return a non null ModelBindingResult but a null model.
+        [Theory(Skip = "ModelBindingResult should be non null if a model binder returns a non null resul #2473.")]
```

Comment:
the only meaning of a non-`null` `ModelBindingResult` with `IsModelSet == false` is a signal _to_ `CompositeModelBinder`.  this test's expectations are thus rather strange.